### PR TITLE
Cleanup/input file

### PR DESCRIPTION
**Pull request by jgiven100**
Update Details
Here is the existing input file documentation as single .md file. Lets decided where we actually want to put this information (either .md file or github wiki).

Also, lets added the new features (implicit & 2-phase) to our documentation before merging (or generating wiki).

Linked Issue
https://github.com/geomechanics/mpm/issues/22

### CODE REVIEW

File: INPUT.md [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+
+### VTK
+
+Available VTK files include `stresses`, `strains`, `displacements`, and `velocities`.
+
+### VTK State Variables
+
+State variables are generated based on their phase ID (e.g., solid is `0`, liquid is `1`). Available state variables are material dependent. 
+
+
```
Comment (jgiven100) [inline INPUT.md:373]: @bodhinandach I'm not even sure if this is true since I mostly use the HDF5 output. More details might make sense here.

Comment (geomechanics-berkeley) [reply INPUT.md:373]: Yes, it is still used. I am using that all the time

Comment (bodhinandach) [reply INPUT.md:373]: Yes, I am using this all the time. Please keep it this way.

Comment (jgiven100) [reply INPUT.md:373]: That wasn't my questions... I'm wondering what are the options for VTK? (i.e., are there other outputs available?)

**Pull request closed without merging**